src/Core/Domain/Product/Combination/QueryResult/CombinationForEditing.php

```diff
@@ -68,6 +68,11 @@ class CombinationForEditing
      */
     private $imageIds;
 
+    /**
+     * @var bool
+     */
+    private $isDefault;
```

Comment:
Please change the scope to `protected`. The class is inheritable.

Reply:
I get the idea, but all the other properties are private, so if we want to introduce a new concept (probably should have done when decided to get rid of final classes) - we should do it in separate PR's. It doesnt make sense to make single property protected while all others are private (and changing all the proprierties would be out of scope)

Reply:
Yeah it's not consistent to change the scope of variables in this PR, or we should change all the queries and commands if do it here Please open an ADR if you wish to change this, I'm not against the idea but it's an architectural choice that was never discussed

@Progi1984 I don't think should impose it on current PRs and block them just for this until a decision has been made ith the other maintainers